packages/fast-components-react-base/src/tabs/examples.data.tsx

```diff
@@ -252,29 +276,43 @@ const examples: ComponentFactoryExample<TabsHandledProps> = {
     detailData: {
         ...tabsManagedClasses,
         label: "A set of example text content",
-        children: detailChildren,
+        tabItems: detailTabItemData,
+        children: ["child 1", "child 2"],
     },
     data: [
         {
             ...tabsManagedClasses,
-            activeId: "tab04",
+            label: "A set of example text content",
+            tabItems: detailTabItemData,
+            children: ["child 1", "child 2"],
+        },
+        {
+            ...tabsManagedClasses,
             label: "A set of example text content",
             orientation: Orientation.horizontal,
-            children: exampleChildren1,
+            tabItems: exampleTabItemData1,
         },
         {
+            /**
+             * @Deprecated 3.4.0
+             */
             ...tabsManagedClasses,
-            activeId: "tab03",
```

Comment:
why are we removing activeId?

Reply:
I can add the activeId back but it's not breaking when I do.